.flake8

```diff
@@ -0,0 +1,2 @@
+[flake8]
+per-file-ignores = tests/test_app.py:F841
```

Comment:
if you're using ruff, you don't need flake8 too.

Reply:
It has to be fixed otherwise pre-commit will fail. Fixing now and resolving this comment

Reply:
done